src/ophys_etl/modules/suite2p_registration/utils.py

```diff
@@ -3,6 +3,125 @@
 from typing import Tuple, List
 from scipy.ndimage.filters import median_filter
 from ophys_etl.utils.array_utils import normalize_array
+from suite2p.registration.rigid import shift_frame
+
+
+def find_movie_start_end_empty_frames(
+        h5py_name: str,
+        h5py_key: str,
+        n_sigma: float = 5,
+        logger: callable = None) -> Tuple[int, int]:
+    """Load a movie from HDF5 and find frames at the start and end of the
+    movie that are empty or pure noise and 5 sigma discrepant from the
+    average frame.
+
+    If a non-contiguous set of frames is found, the code will return 0 for
+    that half of the movie and throw a warning about the quality of the data.
+
+    Parameters
+    ----------
+    h5py_name : str
+        Name of the HDF5 file to load from.
+    h5py_key : str
+        Name of the dataset to load from the HDF5 file.
+    n_sigma : float
+        Number of standard deviations beyond which a frame is considered an
+        outlier and "empty".
+    logger : callable, optional
+        Function to print warning messages to.
+    """
```

Comment:
Please define the returned tuple

Reply:
Changed to `(n_trim_start, n_trim_end)` and added a little more text.